PROGRESS.md

```
@@ -14,10 +14,11 @@
 | 2   | [phase-actions-system](#2-phase-actions-system)                   | `library/`    | 37        | 37        | **100%** |
 | 3   | [phase-demo-showcase](#3-phase-demo-showcase)                     | `composeApp/` | 36        | 36        | **100%** |
 | 4   | [phase-3-expand-library](#4-phase-3-expand-library)               | `library/`    | 239       | 239       | **100%** |
-| 5   | [phase-2-editor-mvp](#2-phase-2-editor-mvp)                       | `composy/`    | 58        | 19        | **32%**  |
-| 6   | [phase-4-semantics-testability](#3-phase-4-semantics-testability) | `library/`    | 30        | 0         | **0%**   |
-| 7   | [phase-3-differentiators](#4-phase-3-differentiators)             | Multi-module  | 37        | 0         | **0%**   |
-|     | **TOTAL**                                                         |               | **620**   | **518**   | **83%**  |
+| 5   | [phase-1.5-mvi-architecture](#5-phase-15-mvi-architecture)        | `composy/`    | 12        | 0         | **0%**   |
+| 6   | [phase-2-editor-mvp](#6-phase-2-editor-mvp)                       | `composy/`    | 58        | 19        | **32%**  |
+| 7   | [phase-4-semantics-testability](#7-phase-4-semantics-testability) | `library/`    | 30        | 0         | **0%**   |
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)